### PR TITLE
DOCS-2420: Merge autogen: base, board

### PR DESCRIPTION
**Pull request by andf-viam**
WIP

When not WIP (but WIP), this PR https://github.com/viamrobotics/rdk/pull/4132 includes some new info from eng that informs some of the changes in what is included and not included in this API.

### CODE REVIEW

**Comment by npentrel** [comment]
what's still wip on this?

**Comment by andf-viam** [comment]
@npentrel Sorry, marked as ready.

File: docs/components/board/_index.md [hunk 1]
```diff
@@ -99,1209 +99,9 @@ import (
...
 The board component supports the following methods:
 
-{{< readfile "/static/include/components/apis/board.md" >}}
+{{< readfile "/static/include/components/apis/generated/board-table.md" >}}
 
-Additionally, the nested `GPIOPin`, `Analog`, and `DigitalInterrupt` interfaces support the following methods:
-
-[`GPIOPin`](#gpiopin-api) API:
-
-{{< readfile "/static/include/components/apis/gpiopin.md" >}}
```
Comment (npentrel) [inline docs/components/board/_index.md:108]: Should these files also be getting deleted?

File: docs/components/board/_index.md [hunk 2]
```diff
@@ -99,1209 +99,9 @@ import (
...
-
-**Parameters:**
-
-- `interrupts` (List[[DigitalInterrupt](https://python.viam.dev/autoapi/viam/components/board/board/index.html#viam.components.board.board.Board.DigitalInterrupt)]): List of digital interrupts to receive ticks from.
-
-**Returns:**
-
-- [(`TickStream`)](https://python.viam.dev/autoapi/viam/components/board/board/index.html#viam.components.board.board.TickStream): [Stream](https://python.viam.dev/autoapi/viam/streams/index.html#viam.streams.Stream) of [Ticks](https://python.viam.dev/autoapi/viam/proto/component/board/index.html#viam.proto.component.board.StreamTicksResponse), objects containing `pin_name`, `time`, and `high` fields.
-
-```python {class="line-numbers linkable-line-numbers"}
```
Comment (npentrel) [inline docs/components/board/_index.md:438]: This seems to have gone missing?

File: docs/components/board/_index.md [hunk 3]
```diff
@@ -99,1209 +99,9 @@ import (
...
-
-{{% /tab %}}
-{{< /tabs >}}
-
-### Write
-
-Write an analog value to a pin or analog output on a board capable of doing so.
-
-{{< tabs >}}
-{{% tab name="Python" %}}
```
Comment (npentrel) [inline docs/components/board/_index.md:1134]: This went missing I think? Unless no longer in use

**Review by npentrel — APPROVED** [review]
Lgtm % checking the three methods I marked as missing

File: docs/components/board/_index.md (hunk 1 above)
Comment (andf-viam) [reply docs/components/board/_index.md:108]: Thank you, good catch, removed all three I missed:

```
        deleted:    static/include/components/apis/analog.md
        deleted:    static/include/components/apis/digitalinterrupt.md
        deleted:    static/include/components/apis/gpiopin.md
```

File: docs/components/board/_index.md (hunk 3 above)
Comment (andf-viam) [reply docs/components/board/_index.md:1134]: This one is malformed upstream, in a manner that usually reflects purposeful deprecation or an unimplemented method. I'll try to track down what/why after this PR.

**Pull request merged**